### ISSUE TITLE
Impossible to open DevTools of webview

### ISSUE THREAD

**Issue by Sunvas** [nwjs/nw.js#8096]
### Current/Missing Behavior

It is completely impossible to open DevTools of a webview: neither by context menu neither by JavaScript API.

### How to reproduce

package.json
```
{
	"main": "/main.html",
	"name": "demo"
}
```

main.html
```
<!doctype html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<title>Demo</title>
</head>
<body>
	<h6>WebView 1</h6>
	<webview id="webview1" src="https://google.com"></webview>
	
	<h6>WebView 2</h6>
	<webview id="webview2" src="https://gmail.com"></webview>
	
	<button onclick="DevTools1()">OpenDevTools 1</button> <button onclick="DevTools2()">OpenDevTools 2</button>
	<script>
	function DevTools1()
	{
		document.getElementById("webview1").showDevTools(true);
	}
	function DevTools2()
	{
		document.getElementById("webview2").showDevTools(true);
	}
	</script>
</body>
</html>
```

And there is no way to open both DevTools simultaneously. Quite strange behaviour.

### Video
https://youtu.be/O0wJpmhK6X8 
Please note that DevTools is called on one webview and displayed for another.

### Additional Info

- Operating System: Windows
- NW.js Version: 0.78, 0.78.1

**Comment by rogerwang** [comment]
We have test case guarding this feature and it works well: https://github.com/nwjs/nw.js/tree/nw78/test/sanity/webview-cdt

**Comment by Sunvas** [comment]
@rogerwang , in your test you have
```
var demodev = document.getElementById('demodev');
```
which means opening only one DevTools window (of webview with ID `demodev`) . But in my case i'm trying to open 2+ DevTools windows and that's impossible.

**Comment by Sunvas** [comment]
@rogerwang  I've added video.

https://youtu.be/O0wJpmhK6X8
Please note that DevTools is called on one webview and displayed for another.

**Comment by Sunvas** [comment]
0.79 - the same bug.

**Comment by Sunvas** [comment]
Despite it is impossible to open DevTools for specified webview and entire window, nevertheless this message also appears in console:
`DevTools debugger is disabled because it is attached to a process that hosts multiple top-level frames, where DevTools debugger doesn't work properly. Please relaunch the browser with --disable-features=ProcessPerSiteUpToMainFrameThreshold to enable debugger.`

**Comment by rogerwang** [comment]
Try this workaround: run nw with `--disable-features=DevToolsTabTarget`

**Comment by Sunvas** [comment]
That works. It worth to mention this param in docs.

**Comment by rogerwang** [comment]
close as fixed in 0.79.1
